tests/system/robot/chromeTests.robot

```diff
@@ -24,7 +24,7 @@ default teardown
 	quit NVDA
 
 default setup
-	start NVDA	standard-dontShowWelcomeDialog.ini	chrome-gestures.ini
+	start NVDA	standard-dontShowWelcomeDialog.ini

```

Comment:
should the file `chrome-gestures.ini` be deleted?

Reply:
Yes. Done in a200a9c36ca82634bba1a583bcdc4cc741cec198.

It was actually my intention but I have forgotten it.